units/host_monitoring_external_termination.go

```diff
@@ -136,10 +137,19 @@ func handleExternallyTerminatedHost(ctx context.Context, id string, env evergree
 		if cloudStatus != cloud.StatusTerminated && (h.UserHost || h.StartedBy != evergreen.User) {
 			return false, errors.New("non-agent host is not already terminated and should not be terminated")
 		}
-
+		if h.RunningTask != "" {
+			if err := task.AddHostCreateDetails(h.RunningTask, h.Id, errors.New("error adding host create error details")); err != nil {
```

Comment:
Sorry, I think this still isn't right:
The error you're passing into AddHostCreateDetails is the error we'll be returning to the user. The error we want to tell them isn't that we couldn't add host create details (although logging this to grip is reasonable); the thing we _want_ to tell them about is that the host was terminated externally.

Reply:
Ah apologies, had mistaken this error for the error we want to show if we fail to update the details themselves.  Fixed.